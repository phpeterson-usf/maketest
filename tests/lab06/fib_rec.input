$(repo_dir)/fib_rec_main
