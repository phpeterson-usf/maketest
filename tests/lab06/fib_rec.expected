fib_rec_c(0) = 0
fib_rec_c(0) = 0
fib_rec_c(1) = 1
fib_rec_s(1) = 1
fib_rec_c(2) = 1
fib_rec_s(2) = 1
fib_rec_c(3) = 2
fib_rec_s(3) = 2
fib_rec_c(10) = 55
fib_rec_s(10) = 55
fib_rec_c(20) = 6765
fib_rec_s(20) = 6765
